components/eam/src/physics/crm/crm_physics.F90

```diff
@@ -227,7 +227,17 @@ subroutine crm_physics_register()
       call pbuf_add_field('CRM_QT',    'global', dtype_r8,dims_crm_3D,idx)
       call pbuf_add_field('CRM_QP',    'global', dtype_r8,dims_crm_3D,idx)
       call pbuf_add_field('CRM_QN',    'global', dtype_r8,dims_crm_3D,idx)
+      ! CRM effective radii - probably only used by COSP. See note below.
+      call pbuf_add_field('CRM_REL' , 'physpkg', dtype_r8,dims_crm_3D,idx)
```

Comment:
Should we add these lines outside of this if block?
Or will they need to be different variables for P3?

Reply:
Probably should move outside this block. I think it was originally this way because we were computing effective radii differently for 2-moment, but for COSP I think this is fine. Note that this only affects the radar simulator I think, and maybe the lidar simulator.